rust/backend/ebpf/src/jni_references.rs

```diff
@@ -11,10 +12,10 @@ use backend_common::{JNICall, JNIMethodName};
 const MAP_MAX_ENTRIES: u32 = 100;
 
 #[map(name = "JNI_REF_CALLS" )]
-static JNI_REF_CALLS: RingBuf = RingBuf::with_byte_size(MAP_MAX_ENTRIES * mem::size_of::<JNICall>() as u32, 0);
+static JNI_REF_CALLS: RingBuf = RingBuf::pinned(MAP_MAX_ENTRIES * mem::size_of::<JNICall>() as u32, 0);
 
 #[map(name = "JNI_REF_PIDS")]
-static JNI_REF_PIDS: HashMap<u32, u32> = HashMap::with_max_entries(4096, 0);
+static JNI_REF_PIDS: HashMap<u32, u64> = HashMap::pinned(4096, 0);
```

Comment:
I did it because on the userspace it was already assumed that it would be u64 which would have crashed.

I could have also changed the userspace to u32, but u64 seemed more consistent.